pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -54,6 +54,7 @@ type EventListener struct {
 type EventListenerSpec struct {
 	ServiceAccountName string                 `json:"serviceAccountName"`
 	Triggers           []EventListenerTrigger `json:"triggers"`
+	Public             bool                   `json:"public"`
```

Comment:
This is not necessary anymore since we are using Ingress right?

Reply:
Great catch!